src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -202,8 +214,8 @@ private static void executeSqlQuery(String sqlQuery) throws SQLException {
         }
     }
 
-    private static void dropAllTables() throws SQLException {
-        executeSqlQuery("DROP SCHEMA public CASCADE");
-        executeSqlQuery("CREATE SCHEMA public");
+    private static void dropTables(String... tableNames) throws SQLException {
+        String tableNamesAsString = String.join(",", tableNames);
```

Comment:
Cool, I did not know that it was possible

Reply:
I also didn't) I saw usage of this  method in Ivan's code and was also surprised) I also find out that this method is very useful in some cases. Pay your attention that it works since Java8...